packages/components/psammead-podcast-promo/src/__snapshots__/index.test.jsx.snap

```diff
@@ -370,25 +370,30 @@ exports[`Podcast Promo Examples basic example 1`] = `
   z-index: 1;
 }
 
-.emotion-14:visited .podcast-promo--visited {
+.emotion-12:visited .podcast-promo--visited {
   color: #AEAEB5;
 }
 
-.emotion-14:focus .podcast-promo--focus {
+.emotion-12:focus .podcast-promo--focus {
   -webkit-text-decoration: underline;
   text-decoration: underline;
 }
 
-.emotion-22 > svg {
-  width: 16px;
-  height: 16px;
+.emotion-20 {
+  color: #6E6E73;
+}
+
+.emotion-20 > svg {
   fill: currentColor;
+  color: unset;
+  width: 1rem;
+  height: 1rem;
   position: relative;
   bottom: 2px;
```

Comment:
Just checking we had inteded a px value here and for right?

Reply:
This is another instance of using REM to size an SVG rather than pixels, so that it resizes based on the html base font size.

Reply:
These are just visually offsetting the SVG slightly to get its alignment correct. I've changed them to REM so that they scale with browser font size changes